internal/project/common/image.go

```diff
@@ -52,7 +52,7 @@ func NewImage(meta *meta.Options, name string) *Image {
 // CompileDrone implements drone.Compiler.
 func (image *Image) CompileDrone(output *drone.Output) error {
 	output.Step(drone.MakeStep(image.Name()).
-		DependsOn(dag.GatherMatchingInputNames(image, dag.Implements[*drone.Compiler]())...),
```

Comment:
this change is not clear to me - graph nodes should be pointers

Reply:
`drone.Compiler` is an interface so we actually do not want pointer here. The reason we used it before, because reflect do not understands interfaces but do understand pointers to the interfaces.

Reply:
ah, makese sense, thanks!